### ISSUE TITLE
tideways.so has been renamed tideways_xhprof.so

### ISSUE THREAD

**Issue by oxyc** [geerlingguy/ansible-role-php-tideways#3]
I just rebuilt a VM and received the following message:

> TASK [geerlingguy.php-tideways : Move Tideways module into place.] *************
fatal: [mush]: FAILED! => {"changed": true, "cmd": "cp /root/php7.1/php-profiler-extension-master/modules/tideways.so /usr/lib/php/7.1/modules/tideways.so", "delta": "0:00:00.005509", "end": "2018-01-02 17:26:17.416453", "failed": true, "msg": "non-zero return code", "rc": 1, "start": "2018-01-02 17:26:17.410944", "stderr": "cp: cannot stat '/root/php7.1/php-profiler-extension-master/modules/tideways.so': No such file or directory", "stderr_lines": ["cp: cannot stat '/root/php7.1/php-profiler-extension-master/modules/tideways.so': No such file or directory"], "stdout": "", "stdout_lines": []}

```
root@mush:~# ls /root/php7.1/php-profiler-extension-master/modules/
tideways_xhprof.so
```

Upstream commit: https://github.com/tideways/php-profiler-extension/commit/fc808db5c3df0e4055db612d15224b3acd1eaa88#diff-cc84bd388da4da3d60ce0f10c2250a42

**Comment by oxyc** [comment]
So this actually requires more work for the tideways paid service:

> This repository now contains an extension by the name of tideways_xhprof, which only contains the XHProf related (Callgraph) Profiler functionality.
>
> Previously the tideways extension contained this functionality together with other functionality used in our Software as a Service.

**Comment by fubarhouse** [comment]
Also affected.

I decided to try this out via DrupalVM, however the provision fails on this task and the role is basically unusable (for me) in its current state.

**Comment by geerlingguy** [comment]
Gah, yeah, same here.

**Comment by geerlingguy** [comment]
Fixed, now it's in version 2.0.0.
